stress_test/run.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+source "utils/cachengo.sh"
+
+function do_install {
+  set -e
+  cachengo-cli updateInstallStatus $APPID "Installing"
+
+  if [ "$SERVER" == "true" ]; then
+    cachengo-cli updateInstallStatus $APPID "Installing: Network Stress Test"
+    apt install iperf -y
+    cp stress_test/stress_network_server.service /lib/systemd/system/
+    cachengo-cli updateInstallStatus $APPID "Installing: CPU Stress Test"
+    apt install stress -y
+    cp stress_test/stress_cpu.service /lib/systemd/system/
+    cachengo-cli updateInstallStatus $APPID "Installing: GPU Stress Test"
+    apt install hashcat -y
+    cp stress_test/stress_gpu.service /lib/systemd/system/
+    cd / && curl -L https://github.com/hashcat/hashcat/archive/refs/tags/v5.1.0.zip -o hashcat-5.1.0.zip && cd -
+   # cp stress_test/hashcat-5.1.0.7z /
+   # apt install p7zip -y
+   # p7zip -d /hashcat-5.1.0.7z
+    unzip /hashcat-5.1.0.zip -d /
+    cachengo-cli updateInstallStatus $APPID "Installing: Starting Stress Test"
+    systemctl enable stress_network_server.service stress_cpu.service stress_gpu.service
+    systemctl daemon-reload
+    systemctl start stress_network_server.service stress_cpu.service stress_gpu.service
```

Comment:
We can probably move all of this outside of the if statement to reduce redundancy